qiita_plugins/biom_type/biom_type_plugin/summary.py

```diff
@@ -6,8 +6,19 @@
 # The full license is in the file LICENSE, distributed with this software.
 # -----------------------------------------------------------------------------
 
+import pandas as pd
 
-def generate_html_summary(qclient, job_id, parameters, out_dir):
+from biom import load_table
+from urllib import quote
+from base64 import b64encode
+from os.path import join, basename
+
+from StringIO import StringIO
+import seaborn as sns
+
+
+def generate_html_summary(qclient, job_id, parameters, out_dir,
+                          return_html=False):
```

Comment:
I think this is not needed for testing, as the file is not moved during the testing procedure and the output filepath can be deterministically known where it is going to be.